code/game/objects/items/holy_weapons.dm

```diff
@@ -572,6 +597,7 @@
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb_continuous = list("smashes", "bashes", "hammers", "crunches")
 	attack_verb_simple = list("smash", "bash", "hammer", "crunch")
+	menu_description = "A war hammer. Capable of tapping knees to measure brain health. Can be worn on the belt."
```

Comment:
it can also be used for kneecapping

Reply:
#59816 added kneecapping only for the baseball bats so far, so I would not be so sure about that. The only element that is added in Initialize() is just kneejerking.